client/src/index.js

```diff
@@ -249,26 +210,28 @@ function createFS(options) {
 
     sync.state = sync.SYNC_DISCONNECTED;
     sync.emit('disconnected');
-    log.info('Disconnected');
+    log.info('Disconnected from MakeDrive server');
   };
 
   // Request that a sync begin.
+  // sync.request does not take any parameters
+  // as the path to sync is determined internally
   sync.request = function() {
-    // sync.request does not take any parameters
-    // as the path to sync is determined internally
-    // requestSync on the other hand optionally takes
-    // a path to sync which can be specified for
-    // internal use
-    requestSync();
+    if(sync.state === sync.SYNC_CONNECTING || sync.state === sync.SYNC_DISCONNECTED) {
+      sync.emit('error', new Error('MakeDrive error: MakeDrive cannot sync as it is either disconnected or trying to connect'));
```

Comment:
Let's stop doing this.  These aren't really errors, and it makes the client code crappy for users, since their console fills with all kinds of warnings.  We should probably just silently return here, or maybe just log.warn.